### PR TITLE
[stable/goldilocks] Version Bumps, probe fix when base path set

### PR DESCRIPTION
**Pull request by lukerogers**
**Why This PR?**
1. The base path functionality for [#185](https://github.com/FairwindsOps/goldilocks/issues/185) hasn't been released yet, this assumed it will go out in 4.3.4, so this shouldn't be merged until then
2. The path to the probe was incorrect (my bad), this fixes previous PR #828 

**Changes**
Changes proposed in this pull request:
*Add logic to ensure the base path has correct formatting for health probe

**Checklist:**

* [ X ] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [ X ] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [ X ] Any new values are backwards compatible and/or have sensible default.
* [ X ] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

### CODE REVIEW

File: stable/goldilocks/templates/dashboard-deployment.yaml [hunk 1]
```diff
@@ -74,11 +74,11 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: {{.Values.dashboard.basePath | default "/" }}health
+              path: {{ include "goldilocks.basePath" . }}/health
```
Comment (rbren) [inline stable/goldilocks/templates/dashboard-deployment.yaml]:
```suggestion
              path: {{ trimSuffix "/" .Values.dashboard.basePath }}/health
```

I think this is simpler (no need for the template). It's also closer to the current logic (which expects a `/` prefix for this value)

File: stable/goldilocks/templates/dashboard-deployment.yaml [hunk 2]
```diff
@@ -74,11 +74,11 @@ spec:
...
               port: http
           readinessProbe:
             httpGet:
-              path: {{.Values.dashboard.basePath | default "/" }}health
+              path: {{ include "goldilocks.basePath" . }}/health
```
Comment (rbren) [inline stable/goldilocks/templates/dashboard-deployment.yaml]:
```suggestion
              path: {{ trimSuffix "/" .Values.dashboard.basePath }}/health
```

Comment (lukerogers) [reply stable/goldilocks/templates/dashboard-deployment.yaml]: That is way better, thanks for the suggestion!

**Comment by sudermanjr** [comment]
My apologies @lukerogers , I completely missed this and may have actually fixed in another PR. It does seem like your implementation may better though, so if you're willing to rebase, I think we can get your contribution in.

**Pull request closed without merging**